src/hexen/mn_menu.c

```diff
@@ -1779,10 +1779,36 @@ void MN_DeactivateMenu(void)
 
 void MN_DrawInfo(void)
 {
+    lumpindex_t lumpindex; // [crispy]
+
     I_SetPalette(W_CacheLumpName("PLAYPAL", PU_CACHE));
-    V_CopyScaledBuffer(I_VideoBuffer,
-           (byte *) W_CacheLumpNum(W_GetNumForName("TITLE") + InfoType,
-                                   PU_CACHE), ORIGWIDTH * ORIGHEIGHT);
+    // V_CopyScaledBuffer(I_VideoBuffer,
+    //        (byte *) W_CacheLumpNum(W_GetNumForName("TITLE") + InfoType,
+    //                                PU_CACHE), ORIGWIDTH * ORIGHEIGHT);
+
+    // [crispy] Refactor to allow for use of V_DrawFullscreenRawOrPatch
+
+    switch (InfoType)
```

Comment:
I like that the original code requires no prior knowledge of the names of the next lumps after "TITLE". Do you think we could keep it that way?

Reply:
The original approach works with the lumps out of hexen.wad but it fails when using replacement graphics from a pwad. (Example included) The original code is dependent on the four lumps being grouped together and having consecutive indices.

[hexen_ws.zip](https://github.com/fabiangreffrath/crispy-doom/files/7882883/hexen_ws.zip)